README.md

```diff
@@ -106,30 +106,24 @@ If you plan to develop with TorchServe and change some of the source code, you m
     sudo apt-get install openjdk-11-jdk
     ```
 
-1. Install dependencies
-
-    ```bash
-    pip install psutil future
```

Comment:
You might need `psutil` to build. [code reference](https://github.com/pytorch/serve/blob/master/ts/metrics/process_memory_metric.py#L10) . I think the above code reference will be invoked when running torch-serve.

Reply:
This is added as a dep inside setup.py. This way the conda binaries which are built out of pip won't have to install the deps separately.